src/Rule.js

```diff
@@ -87,7 +87,11 @@ const Rule = Orderable(
       }
 
       if (!omit.includes('test') && 'test' in obj) {
-        this.test(obj.test instanceof RegExp ? obj.test : new RegExp(obj.test));
+        this.test(
+          obj.test instanceof RegExp || obj.test instanceof Function
```

Comment:
Can you change `obj.test instanceof Function` to `typeof obj.test === 'function'`?